_data/developer.yml

```diff
@@ -310,12 +320,14 @@ websites:
     - name: StatusPage.io
       url: https://www.statuspage.io
       twitter: statuspageio
+      facebook: statuspageio
       img: statuspageio.png
       tfa: No
 
     - name: Taiga.io
       url: https://taiga.io
       twitter: taigaio
+      facebook: taigaio
```

Comment:
Their last post appears to be from November 2015 and they do not post often. Should this still be added?